toxic_timeout.go

```diff
@@ -8,6 +11,10 @@ type TimeoutToxic struct {
 	Enabled bool `json:"enabled"`
 	// Times in milliseconds
 	Timeout int64 `json:"timeout"`
+	// If true, use Toxicity
+	SometimesToxic bool `json:"sometimesToxic"`
+	// 'Toxicity' or probability of timing out 0..1
+	Toxicity float32 `json:"toxicity"`
```

Comment:
Why have both `SometimesToxic` and `Toxicity`? Toxicity could just default to `1` (0 doesn't make sense).